NEWS

```diff
@@ -1,5 +1,23 @@
 systemd System and Service Manager
 
```

Comment:
Please add a `CHANGES WITH 253 in spe:` line above

Reply:
Done.